tests/test_ordering.py

```diff
@@ -149,7 +147,9 @@ def test_6(): pass
     def test_7(): pass
     """
 
-    assert item_names_for(tests_content) == ['test_7', 'test_3', 'test_1', 'test_2', 'test_5', 'test_4', 'test_6']
+    assert item_names_for(tests_content) == ['test_7', 'test_3', 'test_1',
+                                             'test_2', 'test_5', 'test_4',
+                                             'test_6']
 
```

Comment:
The formatting is due to automatic PEP-8 format - this was unintentional, can revert this.